WordPress/Classes/ViewRelated/Blog/SharingViewController.m

```diff
@@ -358,11 +357,12 @@ - (void)syncSharingButtonsIfNeeded
 {
     // Sync sharing buttons if they have never been synced. Otherwise, the
     // management vc can worry about fetching the latest sharing buttons.
-    SharingService *sharingService = [[SharingService alloc] initWithManagedObjectContext:[self managedObjectContext]];
-    NSArray *buttons = [sharingService allSharingButtonsForBlog:self.blog];
+    NSArray *buttons = [SharingButton allSharingButtonsForBlog:self.blog inContext:[self managedObjectContext] error:nil];
```

Comment:
Would it be worth to pass an error pointer here, so that we can log and bail if the method throws?